components/TokenGenerator.php

```diff
@@ -6,50 +6,61 @@
      * Class for creating tokens. Will be used later to create tokens for user connections
      */
     class TokenGenerator {
+
+        /**
+         * Key for generating only token with symbols from [0-9] range
+         * 
+         * @var string
+         */
+        public const DIGITS = "digits";
+
         /**
          * Symbols used to create a token
          *
          * @var string[]
          */
         private $symbols = [];
-        /**
-         * Length of the symbols array
-         * 
-         * @var int
-         */
-        private $length;
```

Comment:
Length property is no longer needed because array element is being selected with array_rand function.